test/podman_run_security.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+    cleanup_test
+}
+
+function setup() {
+    copy_images
+}
+
+@test "run privileged test" {
+    cap=$(grep CapEff /proc/self/status | cut -f2 -d":")
+
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} run --privileged ${ALPINE}  grep CapEff /proc/self/status
```

Comment:
Any reason why no doing bash -c here?

Reply:
I am actually running the grep command in the container, as opposed to piping the output to the grep command.  Bottom line, no reason to do it.

Reply:
Then aren't you doing the same thing for the 2 tests below? No point in have bash -c there as well.